source/common/secret/secret_manager_impl.cc

```diff
@@ -34,6 +34,9 @@ void SecretManagerImpl::addStaticSecret(const envoy::api::v2::auth::Secret& secr
     }
     break;
   }
+  // Static secret of type
```

Comment:
no need this comment, CombinedValidationContext is not in oneof in Secret

Reply:
Removed. Thanks!